packages/connectivity/connectivity/example/lib/main.dart

```diff
@@ -122,6 +122,9 @@ class _MyHomePageState extends State<MyHomePage> {
         } on PlatformException catch (e) {
           print(e.toString());
           wifiName = "Failed to get Wifi Name";
+        } on MissingPluginException catch (e) {
+          print(e.toString());
+          wifiName = "Location permissions not enabled";
```

Comment:
This should be part of a debug message and not the wifiName, which someone could interpret as a valid name. But then they do use it in this way on L146.

So I vote to keep it as is and see what they say in review.